src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLPatternImpl.java

```diff
@@ -249,8 +247,14 @@ private List<Trip> getTrips(DataFetchingEnvironment environment) {
     return getSource(environment).scheduledTripsAsStream().collect(Collectors.toList());
   }
 
-  private RoutingService getRoutingService(DataFetchingEnvironment environment) {
-    return environment.<LegacyGraphQLRequestContext>getContext().getRoutingService();
+  private RealtimeVehiclePositionService getVehiclePositionsService(
+    DataFetchingEnvironment environment
+  ) {
+    return environment
+      .<LegacyGraphQLRequestContext>getContext()
+      .getServerContext()
+      .graph()
+      .getVehiclePositionService();
```

Comment:
I'd add this to LegacyGraphQLRequestContext

Reply:
Done.